src/rocker/core.py

```diff
@@ -251,12 +251,24 @@ def docker_build(docker_client = None, output_callback = None, **kwargs):
         print("no more output and success not detected")
         return None
 
-def docker_remove_image(image_id, docker_client = None, output_callback = None, **kwargs):
+def docker_remove_image(
+        image_id,
+        docker_client = None,
+        output_callback = None,
```

Comment:
This is kind of drive-by but `output_callback` appears completely unused. Is it deprecated?

Reply:
Yeah, this doesn't have the streaming interface so removing it makes sense.